alchemy-web/src/content/docs/guides/cloudflare-tanstack-start.mdx

```diff
@@ -238,4 +236,36 @@ The `alchemy.run.ts` script will be run by `node` but still needs to infer the [
   },
   "include": ["types/**/*.ts", "alchemy.run.ts", "src/**/*.ts"]
 }
-``` 
\ No newline at end of file
+``` 
+
+## `vite.config.ts`
+
+Use the `alchemyDevEnvironmentShim()` plugin to configure TanStack Start for local development:
+
+```ts
+import { tanstackStart } from "@tanstack/react-start/plugin/vite";
+import viteReact from "@vitejs/plugin-react";
+import alchemyDevEnvironmentShim from "alchemy/cloudflare/tanstack-start";
+import { defineConfig } from "vite";
+import tsConfigPaths from "vite-tsconfig-paths";
+
+export default defineConfig({
+  build: {
+    target: "esnext",
+    rollupOptions: {
+      external: ["node:async_hooks", "cloudflare:workers"],
+    },
+  },
+  plugins: [
+    alchemyDevEnvironmentShim(),
```

Comment:
Do they not need to use our vite plugin too?

Can our plugin include the shim automatically so there's only one.